samples/security/protection_psoc6/README.rst

```diff
@@ -0,0 +1,112 @@
+.. _prot-psoc6-sample:
```

Comment:
Because you're creating the first sample in the ``samples/security`` folder, you'll also need to add another organizational file as found in the other sample folders.  Add a file ``samples/security/security.rst`` with the contents:
```
.. _security-samples:

Security Samples
################

.. toctree::
   :maxdepth: 1
   :glob:

   **/*
```

And edit ``samples/samples.rst`` to reference this new file by adding a line at the end of the toctree list there:
```
   security/security.rst
```